src/hosting/normalizedHostingConfigs.ts

```diff
@@ -56,6 +56,41 @@ function filterOnly(configs: HostingConfig[], onlyString: string): HostingConfig
   return filteredConfigs;
 }
 
+function filterExcept(configs: HostingConfig[], exceptString: string): HostingConfig[] {
+  if (!exceptString) {
+    return configs;
+  }
+
+  const exceptTargets = exceptString.split(",");
+  if (exceptTargets.includes("hosting")) {
+    return [];
+  }
+
+  const exceptSet = new Set(
+    exceptTargets.filter((t) => t.startsWith("hosting:")).map((t) => t.replace("hosting:", ""))
+  );
+
+  const configsBySite = new Map<string, HostingConfig>();
```

Comment:
These are unused, no?  it looks like you just add configs to these and then forget about them...

Reply:
You're right. I added the set to get rid of these and didn't get rid of them